jumpscripts/maintenance/image_update_version.py

```diff
@@ -0,0 +1,40 @@
+import time
+from JumpScale import j
+
+descr = """
+Checks the last modified timestamp of all images and compare it with the available image in the server and update the image if needed.
+"""
+
+organization = "greenitglobe"
+category = "monitor.maintenance"
+author = "ali.chaddad@gig.tech"
+license = "bsd"
+version = "1.0"
+period = "0 0 * * *"  # Once a day
+startatboot = True
+enable = True
+queue = "process"
+roles = ["storagedriver"]
```

Comment:
maybe better to do this on the controller ? (one per grid not many per grid doing the same thing)